frontend/src/components/About.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import AppPageHeader from "./AppPageHeader";
+import PageContainer from "./PageContainer";
+import PageContent from "./PageContent";
+import {Card, Col, Row} from "antd";
+
+const REPOSITORY = "https://github.com/c3g/freezeman"
+
+
+const COL_LAYOUT = {
+  lg: 8,
+  xs: 24,
+  style: {marginTop: "16px", marginRight: "12px"}
+};
+
+
+const About = () => {
+    const commit = GIT_COMMITHASH
+    const branch = GIT_BRANCH
+    const lastUpdate = GIT_LASTUPDATE
+    const version = FMS_VERSION
+    const env = FMS_ENV
+
+    const commitUrl = `${REPOSITORY}/commit/${commit}`
+    const branchUrl = `${REPOSITORY}/tree/${branch}`
+    const shortCommit = commit.slice(0,8)
+
+    return <PageContainer>
+        <AppPageHeader title="About" />
+        <PageContent style={{padding: "0 24px 24px 24px"}}>
+            <Row>
+                <Col {...COL_LAYOUT}>
+                    <Card title="Information" size="large">
+                      FreezeMan
+                      <br/>
+                      {(version ? `Version ${version}` : '')}
+                      <br/>
+                      Released under GNU LGPL version {version} © C3G, McGill University
```

Comment:
`{version}` comes from the freezeman version, but this is license version (`GNU LGPL v3`) and is constant.